README.md

```diff
@@ -5,7 +5,7 @@ Teach nodenv to use .nvmrc in addition to .node-version.
 [![Tests](https://img.shields.io/github/actions/workflow/status/nodenv/nodenv-nvmrc/test.yml?label=tests&logo=github)](https://github.com/nodenv/nodenv-nvmrc/actions/workflows/test.yml)
 [![Latest GitHub Release](https://img.shields.io/github/v/release/nodenv/nodenv-nvmrc?label=github&logo=github&sort=semver)](https://github.com/nodenv/nodenv-nvmrc/releases/latest)
 [![Latest Homebrew Release](<https://img.shields.io/badge/dynamic/regex?label=homebrew-nodenv&logo=homebrew&logoColor=white&url=https%3A%2F%2Fraw.githubusercontent.com%2Fnodenv%2Fhomebrew-nodenv%2Frefs%2Fheads%2Fmain%2FFormula%2Fnodenv-nvmrc.rb&search=archive%2Frefs%2Ftags%2Fv(%3F%3Cversion%3E%5Cd%2B.*).tar.gz&replace=v%24%3Cversion%3E>)](https://github.com/nodenv/homebrew-nodenv/blob/main/Formula/nodenv-nvmrc.rb)
-<!-- [![Latest npm Release](https://img.shields.io/npm/v/@nodenv/nodenv-nvmrc?logo=npm&logoColor=white)](https://www.npmjs.com/package/@nodenv/nodenv-nvmrc/v/latest) -->
+[![Latest npm Release](https://img.shields.io/npm/v/@nodenv/nodenv-nvmrc?logo=npm&logoColor=white)](https://www.npmjs.com/package/@nodenv/nodenv-nvmrc/v/latest)
```

Comment:
This badge will error until we publish to npm (which will be handled later when the nodenv release workflow is added.)